packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -3219,18 +3242,19 @@ export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
     if (inputString === '') {
       return;
     }
+    const selectedColumn = IrisGridUtils.getColumnByName(
+      model.columns,
+      selectedColumnName
+    );
 
-    const columnIndex = model.getColumnIndexByName(selectedColumnName);
-    if (columnIndex === undefined) {
+    if (selectedColumn === undefined) {
       return;
     }
 
-    const selectedColumn = model.columns[columnIndex];
-
     let searchFromRow;
 
     if (this.grid) {
-      ({ selectionEndRow: searchFromRow } = this.grid.state);
+      ({ cursorRow: searchFromRow } = this.grid.state);
```

Comment:
Again if there's no grid, we shouldn't really be doing anything at all since it's not rendered yet

I also saw Bender commented on this and said you can change it to use `this.grid?.state.cursorRow` instead. Why not use that syntax here?

Reply:
Assuming we already know the column and data, and we are just waiting on grid rendering, then I think it's fine if we call seekrow in advance, and render grid with the correct row selected. Thoughts?

Reply:
`grid` should basically always exist except on initial render of the component